discovery-provider/src/tasks/index_aggregate_user.py

```diff
@@ -10,7 +10,10 @@
 # Names of the aggregate tables to update
 AGGREGATE_USER = "aggregate_user"
 DEFAULT_UPDATE_TIMEOUT = 60 * 30  # 30 minutes
-REFRESH_COUNTER = 100
+
+# every ~10000 updates, refresh the entire table
```

Comment:
is this every 10000 updates or every 1000 updates? `REFRESH_COUNTER` seems to be `1000`. Also where is the 30s interval? (Just trying to get context.)

Reply:
oops fixed my comment typo. the interval is here https://github.com/AudiusProject/audius-protocol/blob/6c30c5639ebd7603709a031b0646fb7f1997bffb/discovery-provider/src/app.py#L445.